contracts/Removal.sol

```diff
@@ -284,6 +290,32 @@ contract Removal is
     super.safeBatchTransferFrom(from, to, ids, amounts, projectId);
   }
 
+  /**
+   * @notice Marks an amount of a removal as released given a removal ID and a quantity.
+   * @param removalId The ID of the removal to mark as released.
+   * @param amount The amount of the removal to release.
+   *
+   * ##### Requirements:
+   *
+   * - The contract must not be paused.
+   * - The caller must have the `RELEASER_ROLE`.
+   * - The `amount` for the removal must be <= the removal's initial amount
+   */
+  function release(
+    address owner,
+    uint256 removalId,
+    uint256 amount
+  ) external whenNotPaused onlyRole(RELEASER_ROLE) {
```

Comment:
we probably want to emit events from here too

Reply:
Now that the only time a removal is sent to the 0 address is when releasing happens, i think it's appropriate to just rely on `TransferSingle`/`TransferBatch` where `to == address(0)`. What do you think @amiecorso ?

Reply:
I think that makes sense.  since those events also include the `from` being burned from, we can use that to be able to see whether it's from the market, from certificates, or from unlisted balance that belonged to the original supplier.  I do think that we will probably want to emit events from the `Certificate.releaseRemoval` function that indicate which certificate ids are having some removal id released, because that level of granularity won't be captured by the `TransferSingle`/`TransferBatch` events since the Certificate contract itself is the one that maintains the bookkeeping at that granularity.